agent/acs/handler/acs_handler.go

```diff
@@ -424,9 +423,8 @@ func acsWsURL(endpoint, cluster, containerInstanceArn string, taskEngine engine.
 func newDisconnectionTimer(client wsclient.ClientServer, timeout time.Duration, jitter time.Duration) ttime.Timer {
 	timer := time.AfterFunc(utils.AddJitter(timeout, jitter), func() {
 		seelog.Warn("ACS Connection hasn't had any activity for too long; closing connection")
-		closeErr := client.Close()
-		if closeErr != nil {
-			seelog.Warnf("Error disconnecting: %v", closeErr)
+		if err := client.Close(); err != nil {
```

Comment:
There is only one return value.. so is this still more idiomatic than:
`if client.Close() != nil {`

Reply:
how is that? I don't see how `err := something; if err != nil {}` is more idiomatic than `if err := something; err != nil {}`

Reply:
Right! I was asking if setting the `err` variable is necessary at all.  On closer inspection I see that we log the error, so you should ignore this :)